packages/bridge/contracts/l2/L2DaiGateway.sol

```diff
@@ -135,7 +114,7 @@ contract L2DaiGateway is L2CrossDomainEnabled {
       res
     );
 
-    emit WithdrawalInitiated(l1Token, from, to, id, currExitNum, amount);
+    emit WithdrawalInitiated(l1Token, from, to, id, 0, amount);
```

Comment:
I would add a comment here about the not needed exitNum, but the same event used to keep the event signature